### DISCUSSION TITLE
How to migrate an account?

### DISCUSSION THREAD

**Discussion by Mintoo200** [conference-hall/conference-hall#287] [Q&A] [answered]
I created an account some time ago using my professional Google account. Is there a way to migrate from Google SSO to regular email and password. And, in so doing, a way to change the **login** email (in contrast with the profile email, which I already updated).

Thanks

**Comment by bpetetot** [comment, marked as answer]
Hello @Mintoo200, the email/password authentication method is not yet available.

But it is possible to link a new SSO authentication method (e.g. a personal Google, Twitter or GitHub account) to your existing account. You'll then be able to log in and retrieve your existing data.

You can't do this yourself at the moment, it has to be done by support. Please contact me on support@conference-hall.io and we'll explain how to link your account to a new authentication method. Thanks.

**Comment by erwanosouf** [comment]
Hi @bpetetot !

I am in the same situation (Professional Account, login with Google).
I would like to use GitHub for login in order to keep access to my Conference Hall account for personal use.
However, I already created another (empty) account with GitHub, which prevents me to link the first account.
I did not find a way to delete the second account.

Is there a way I can fix this situation ?

**Comment by bpetetot** [comment]
@erwanosouf 
It should be good now. Your account with GitHub auth has been deactivated.
You can connect to your account with your Google auth, then link your GitHub auth with it.

**Reply by erwanosouf** [reply]
Worked like a charm, thanks!
